### DISCUSSION TITLE
Connecting PLC

### DISCUSSION THREAD

**Discussion by EldenZuken** [MikaylaFischler/cc-mek-scada#250] [General]
I am having issues connecting the PLC to the reactor. It is not showing green when connected via cable and touch contact. The network light is also grey and I am unsure how to make it connect. Two possible ideas I had were 1. I don't have a turbine and boiler attached to the reactor as I don't want to build until I know it all works. 2. I only have 1 reactor and not 4. Got any answers???

**Comment by MikaylaFischler** [comment]
If you have it connected via wired modem, then you would need one wired modem touching the computer, then a cable connecting it to the other modem touching the reactor, then you would need to right click the reactor's wired modem so ComputerCraft connects it to the network (it will get a red border). Was that last step the one that may have been missed?

**Reply by EldenZuken** [reply]
it was red when connected with the wired modem.

**Reply by MikaylaFischler** [reply]
Right clicking the physical modem in-world will make its border red, I don't mean the light on the screen. When you right click it/interact it will say the name of the device in chat, then things should work [https://www.computercraft.info/wiki/Wired_Modem](url)

**Reply by EldenZuken** [reply]
![screenshot1](https://github.com/MikaylaFischler/cc-mek-scada/assets/118336136/742e096b-3c3c-4479-8fa6-4b1a6b78e2cd)
![screenshot](https://github.com/MikaylaFischler/cc-mek-scada/assets/118336136/f8bb1703-722a-4358-8215-a6fe94867c25)

**Reply by EldenZuken** [reply]
I had clicked it.

**Reply by MikaylaFischler** [reply]
Thanks for the pictures, that helps a lot. Can you run for me `peripherals` in the computer's terminal, and what version of Mekanism is on your server?

And apologies if any of my comments sounded repetitive, I'm just trying to clarify all common miscommunications I've had one by one until I get the YouTube setup guides created.

**Reply by EldenZuken** [reply]
Seems that it was the version the mod pack said it was 10.1 while the files were 10.0 testing in a new world as when I changed the version it corrupted my other one.

**Reply by EldenZuken** [reply]
It worked! Thank you for your help the simple problems always get me.

**Reply by MikaylaFischler** [reply]
Simple problems tend to be that way, glad it's figured out!
